test/1.4/errors/T_WPRAGMA.dml

```diff
@@ -0,0 +1,34 @@
+/*
+  © 2024 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+// no warning
+/*% COVERITY irrelevant %*/
+
+/// WARNING WPRAGMA
+/*% MYVERYCOOLPRAGMA %*/
+
+/// WARNING WPRAGMA
+/*% 😀 %*/
+
+/// WARNING WPRAGMA
+/*%
+  MYVERYCOOLPRAGMA%*/
+
+// no warning (parses as a pragma)
+/*%
+  COVERITY
+  irrelevant%*/
```

Comment:
Interesting! It's somewhat unclear what line this is attributed to -- perhaps extend the foo/bar/baz case in pragmas/COVERITY with some newlines to illustrate this? e.g., the following should attach `foo` to `x`, right?
```
/*% COVERITY foo %*/
/*%
 COVERITY
  bar
   %/
x();
```

Reply:
Yeah, but experimentation revealed a bug to how I'm handling this.

```
/*% COVERITY foo %*/
/*%
 COVERITY
  bar
   %/ /*% COVERITY baz %*/
 x();
 ```
 will work as expected, but 
 ```
/*% COVERITY foo %*/
/*% COVERITY bar %*/ /*%
 COVERITY
  baz
   %/
 x();
 ```
 will not; in the latter case it'll skip `bar`. I've added a hack to fix that. Honestly, I'm not sure about how _extreme_ cases we should support. We'll likely revisit this; if we were to add a warning pragma like we discussed, we'd likely to change and generalize the line-association mechanism.